src/components/src/B2cHomePage/b2c.home.page.tsx

```diff
@@ -20,29 +20,30 @@
  */
 
 import React from 'react';
-import { ParallaxProvider, Parallax } from 'react-scroll-parallax';
 import { getConfig, IEpConfig } from '../utils/ConfigProvider';
-
-import Carousel from '../Carousel/carousel.homepage';
 import IndiRecommendationsDisplayMain from '../IndiRecommendations/indirecommendations.main';
 import ImageContainer from '../ImageContainer/image.container';
 
 import './b2c.home.page.less';
 
-import homeEspot2 from '../../../images/site-images/home-block-image-3.jpg';
-import homeEspot3 from '../../../images/site-images/home-block-image-2.jpg';
-import homeEspot4 from '../../../images/site-images/home-block-image-1.jpg';
-import homeEspotParallax2 from '../../../images/site-images/parallax-banner-2.jpg';
-import homeEspotParallax1 from '../../../images/site-images/parallax-banner-1.jpg';
+import heroImagePourCoffee from '../../../images/site-images/hero_image_pour_coffee.png';
```

Comment:
These names regress from what we had before. @petro97 recalls how we changed the names to be generic and easier to replace.